DQM/L1TMonitor/interface/L1MUTK.h

```diff
@@ -0,0 +1,94 @@
+#ifndef L1MUTK_H
+#define L1MUTK_H
+
+/*
+ * \file L1MUTK.h
+ *
+ * $Date: 2015/06/16 12:00:00 $
+ * $Revision: 0.03 $
+ * \author S.Baranov
+ *
+*/
+
+// system include files
+#include <memory>
+#include <unistd.h>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DataFormats/Candidate/interface/LeafCandidate.h"
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+
+using namespace reco;
```

Comment:
header files can not have "using namespace ..."
please remove.
One  easy alternative is to get rid of this .h file and put everything in the .cc file